internal/task/service.go

```diff
@@ -0,0 +1,323 @@
+package task
```

Comment:
The service seems to only wrap database operations, so bit of an overkill. Also it's good idea to initialise temporal client in main as other features likely will depend on it, but we can refactor later not to block this